.github/workflows/blossom-ci.yml

```diff
@@ -0,0 +1,90 @@
+# A workflow to trigger ci on hybrid infra (github + self hosted runner)
+name: Blossom-CI
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+      inputs:
+          platform:
+            description: 'runs-on argument'
+            required: false
+          args:
+            description: 'argument'
+            required: false
+jobs:
+  Authorization:
+    name: Authorization
+    runs-on: blossom
+    outputs:
+      args: ${{ env.args }}
+
+    # This job only runs for pull request comments
+    if: github.event.comment.body == '/build'
+    steps:
+      - name: Check user for team affiliation
+        id: teamAffiliation
+        uses: tspascoal/get-user-teams-membership@v2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          username: ${{ github.actor }}
+          team: nixl-maintainer
```

Comment:
are you sure this is the right team name?
https://github.com/orgs/ai-dynamo/teams/nixl-maintainers
i think need to add 's'